src/cli.ts

```diff
@@ -61,7 +64,11 @@ async function main(args: string[] = []): Promise<void> {
       foregroundPath: cliConfig.foregroundPath
     },
     platforms: cliConfig.platforms,
-    force: cliConfig.force
+    force: cliConfig.force,
+    androidOutputPath: (await fse.pathExists(cliConfig.androidOutputPath))
+      ? cliConfig.androidOutputPath
+      : undefined,
```

Comment:
I don't think it makes sense to silently revert to the default if the configured directory doesn't exist. It sounds like it might be confusing for users. We should trust that the user knows what he or she is doing.